src/handlers/assign/auto.ts

```diff
@@ -0,0 +1,73 @@
+import { Context } from "probot";
+import { getBotContext, getLogger } from "../../bindings";
+import { addAssignees } from "../../helpers";
+import { gitLinkedIssueParser } from "../../helpers/parser";
+import { Payload } from "../../types";
+
+// Use `context.octokit.rest` to get the pull requests for the repository
+export const getPullRequests = async (context: Context) => {
```

Comment:
This method gets all the pull requests opened but the function name doesn't make sense

```typescript
export const getPullRequests = async (context: Context, state = "open")

...
const { data: pulls } = await context.octokit.rest.pulls.list({
      owner: payload.repository.owner.login,
      repo: payload.repository.name,
      state,
})
```

We can use this helper function in case we iterate the both opened/closed PRs.